### ISSUE TITLE
maximization not working

### ISSUE THREAD

**Issue by svigerske** [ERGO-Code/HiGHS#209]
When trying to solve max { x : x = 1 } by means of the .lp file
```
\ File written by Highs .lp filereader
maximize
 obj: +1 x1 
st
 con1: +1 x1 = +1
bounds
 x1 free
binary
general
semi
end
```
I seem to get -1 as optimal value:
```
Running HiGHS 1.0.0 [date: 2019-10-18, git hash: 62be5b7]
Copyright (c) 2019 ERGO-Code under MIT licence terms

loadLpFromFile: lp01.lp
SoS section is not currenlty supported by the .lp filereader.LP       : lp01
Rows     : 1
Cols     : 1
Nonzeros : 1

18:31:45 [INFO   ] checkOptions: Options are OK

# Model file
# [type: string, advanced: false, default: ""]
model_file = lp01.lp
18:31:45 [INFO   ] Presolve reductions: columns 0(-1); rows 0(-1) elements 0(-1)
18:31:45 [INFO   ] Solving the presolved LP
18:31:45 [INFO   ] Solving an unconstrained LP with 0 columns
18:31:45 [INFO   ] HiGHS basic solution: Analysis after returning from postsolve
18:31:45 [INFO   ] HiGHS basic solution: Iterations = 0; Objective = -1; Infeasibilities Pr 0(0); Du 0(0); Status: Optimal
18:31:45 [INFO   ] Solving the original LP from the solution after postsolve
18:31:45 [INFO   ] Scaling: Matrix has min(max) values of 1(1) so none performed
18:31:45 [INFO   ] Initial basis condition estimate of 1 is within the tolerance of 1e+14
18:31:45 [INFO   ] Initial basic solution: Objective = -1; Infeasibilities Pr 0(0); Du 0(0); Status: Optimal
18:31:45 [INFO   ] HiGHS basic solution: Analysis after running the simplex solver
18:31:45 [INFO   ] HiGHS basic solution: Iterations = 0; Objective = -1; Infeasibilities Pr 0(0); Du 0(0); Status: Optimal

Run status : Optimal
Iterations : 0
Objective  : -1.000000e+00
Time       : 0.001
Postsolve  : 0
```
When I solve this by hand, I get 1 as optimal value.

I've a similar issue when doing this with the GAMS interface, but there I see "Objective = 1" in the INFO log lines.

**Comment by jajhall** [comment]
The LP file reader needs to change the objective sense when it encounters maximize. I could probably spot how to do this, but it should be easier for Michael to do it.

**Comment by galabovaa** [comment]
@jajhall can you confirm we have fixed this so we can close the issue?

**Comment by jajhall** [comment]
MPS is done - free and fixed format
Need to look at the .lp file reader - Michael's written a new one to handle QP

**Comment by feldmeier** [comment]
I'll double check how the current LP filereader is handling this, and fix it - integrating the new filereader is going to take a bit longer

**Comment by feldmeier** [comment]
So far the LP filereader was negating all coefficients when maximization was specified (I think this is from a time when Highs itself was ignoring objective sense). This should be fixed with #291.
